offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/person/update/UpdatePersonViewModel.groovy

```diff
@@ -32,14 +31,17 @@ class UpdatePersonViewModel extends CreatePersonViewModel implements Resettable{
     @Bindable
     Boolean personUpdated
 
+    @Bindable
+    Boolean affiliationsValid
+
     UpdatePersonViewModel(ResourcesService<Customer> customerService,
                           ResourcesService<ProjectManager> managerResourceService,
                           ResourcesService<Affiliation> affiliationService,
                           EventEmitter<Person> customerUpdate,
                           ResourcesService<Person> personResourceService) {
         super(customerService, managerResourceService, affiliationService, personResourceService)
         this.customerUpdate = customerUpdate
-        affiliationList = new ArrayList<Affiliation>()
+        affiliationList = new ObservableList(new ArrayList<Affiliation>())
```

Comment:
This fixes an oversight where there was an array list assigned to an observable list